src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryFunction.java

```diff
@@ -328,6 +332,10 @@ private RepositoryDirectoryValue.Builder fetchInternal(
         markerData.put("ENV:" + envKey, clientEnvironment.get(envKey));
       }
 
+      for (Table.Cell<RepositoryName, String, RepositoryName> repoMappings : repoMappingRecorder.recordedEntries().cellSet()) {
```

Comment:
Is this in some canonical order?

Reply:
yes, it's sorted. see impl of `recordedEntries`